safekeeper/src/http/routes.rs

```diff
@@ -32,6 +34,7 @@ struct SafekeeperStatus {
 
 /// Healthcheck handler.
 async fn status_handler(request: Request<Body>) -> Result<Response<Body>, ApiError> {
+    check_permission(&request, None)?;
```

Comment:
Probably should remove this line and leave a comment that this route is in `ALLOWLIST`?